app/scripts/controllers/add-events.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (Api, $scope, $routeParams, $location) {
   Api.Patients.get({ id: $routeParams.patientId }, success, error)
 
   $scope.events = [
-    { code: 'sample-event', display: 'Sample Event', formName: 'FormBuilderaddEventSampleEvent' }
+    { code: 'birth', display: 'Birth Registration', formName: 'FormBuilderAddEventBirth' }
```

Comment:
Just a note, this should be 'Birth Notification'. The terminology is interesting here. In CRVS:

* a notification is basically a flag to say a birth happened with some basic info and someone needs to follow up to get a full declaration
* a declaration is the submission of all birth details with mother and father identity document and any other required info
* a registration is when a birth declaration is verified by some official and is considered registered, at this point a birth certificate can be issued